index.js

```diff
@@ -80,6 +81,7 @@ export const Rels = {
 		explicitEvaluation: 'https://activities.api.brightspace.com/rels/explicit-evaluation',
 		feedback: 'https://activities.api.brightspace.com/rels/feedback',
 		feedbackAttachments: 'https://activities.api.brightspace.com/rels/attachments',
+		iteratorEvaluationApplication: 'https://api.brightspace.com/rels/iterator-evaluation-application',
```

Comment:
@amauryperez I just realized this doesn't have the `activities` subdomain - do we want to add that here and on the API side?

Reply:
We can add it